Firmware/conv2str.cpp

```diff
@@ -104,7 +104,7 @@ char *ftostr43(const float &x, uint8_t offset)
 //Float to string with 1.23 format
 char *ftostr12ns(const float &x)
 {
-  long xx = x * 100;
+  int xx = x * 100;
```

Comment:
are you sure the value can never exceed 32K?

Reply:
It could, but then even the old implementation would be broken. The largest number this can render is 9.99 (unsigned). So when you make it *100, it will be a maximum of 999, which is fine for int16_t.